index.html

```diff
@@ -569,18 +569,33 @@ <h3>Use Cases and Requirements</h3>
         </p>
 
         <p>
-A <dfn class="lint-ignore">conforming processor</dfn> is any algorithm realized
-as software and/or hardware that generates or consumes a <a>conforming
-document</a>. Conforming processors MUST produce errors when non-conforming
-documents are consumed.
+A <dfn>conforming processor</dfn> is any algorithm realized as software and/or
+hardware that generates and/or consumes a <a>conforming document</a>. <a>Conforming
+processors</a> MUST produce errors when non-conforming documents are
+encountered.
         </p>
 
         <p>
-This specification includes both required and optional fields. An <a>issuer</a>
-MUST include all required fields. A <a>verifier</a> SHOULD check that each
-required field satisfies the normative requirements for that field.
-Optional fields MAY
-be ignored by <a>issuers</a> and/or <a>verifiers</a>.
+A <dfn class="lint-ignore">conforming issuer processor</dfn> is a <a>conforming
+processor</a> that produces <a>conforming documents</a>. A <a>conforming
+issuer processor</a> MUST include all required fields in the <a>conforming
+documents</a> that it produces. A <a>conforming issuer processor</a> MUST
+secure the <a>conforming documents</a> it produces using a securing mechanism
+as described in Section <a href="#securing-verifiable-credentials"></a>.
+        </p>
+
+        <p>
+A <dfn class="lint-ignore">conforming verifier processor</dfn> is a
+<a>conforming processor</a> that consumes <a>conforming documents</a>. A
+<a>conforming verifier processor</a> SHOULD check that each required field
+satisfies the normative requirements for that field. A <a>conforming verifier
```

Comment:
Above, we state that `<a>Conforming
processors</a> MUST produce errors when non-conforming documents are
encountered`

If you MUST produce errors, then the SHOULD in this line doesn't have any effect. It appears to me that the two lines are at odds. Am I misunderstanding this?

Reply:
```suggestion
<a>conforming verifier processor</a> MUST check that each required field
satisfies the normative requirements for that field. A <a>conforming verifier
```

Reply:
Applied in 857f422125a07216ec4d3b2cd5e14b7ebf4b02c4.